api/validators/FeedbackControllerRequests.ts

```diff
@@ -24,11 +23,6 @@ export class Feedback implements IFeedback {
   type: FeedbackType;
 }
 
-export class SubmitEventFeedbackRequest implements ISubmitEventFeedbackRequest {
```

Comment:
What's this change for?

Reply:
Moved SubmitEventFeedbackRequest from FeedbackControllerRequests to EventControllerRequests